src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -531,6 +545,68 @@ export function updateGeneQuery(
     return updatedQueries;
 }
 
+// This function acts as a toggle. If present in 'geneQueries', the query
```

Comment:
can we move all these new functions into a separate utils file just for SV?  no time like the present to start breaking things up more.